LSPS1/README.md

```diff
@@ -0,0 +1,503 @@
+# LSPS1 Channel Request
+
+| Name    	| channel_request               |
+|---------	|------------------------------	|
+| Version 	| 2                             |
+| Status    | For Implementation            |
+
+
+## Motivation
+
+The goal of this specification is to provide a standardized LSP API for wallets to purchase a channel from the LSP directly.
+
+## Definitions
+
+### Data types
+
+This specification uses data types defined in [LSPS0 Common Schemas](LSPS0/common-schemas.md). The types are referenced in the format of `LSPS0.{type}`.
+
+### Actors
+
+`LSP` is the API provider. `Client` is the client of the API.
+
+### Channel sides
+
+`client_balance` are the funds on the client side. `lsp_balance` are the funds on the LSP side.
+
+### Overprovisioning
+
+The LSP is allowed to overprovision channels/onchain-payments/onchain-fees as long as it benefits the client.
+
+> **Rationale** The lsp may need to "bin" UTXOs.
+
+### Errors
+
+LSP will return errors according to the JSONRPC 2.0 specification (see [LSPS0 Error Handling](https://github.com/BitcoinAndLightningLayerSpecs/lsp/tree/main/LSPS0#error-handling)).
+
+**LSP**
+- MAY return an `code` that is not listed above.
+
+**Client**
+- MUST be able to handle an `code` that is not listed
+  above.
+  - SHOULD report an unrecognized error code simply as
+    "unrecognized error code".
+  - **Rationale** This allows clients written for older versions
+    of this specification to work with LSPs written for newer
+    versions.
+- SHOULD NOT display an unrecognized `code`, `message` or `data` to the user.
+  **Rationale** Unrecognized `code`s may be misinterpreted
+  or misunderstood by users.
+
+Any error MAY include a `message` field that is a human-readable string. The `message` field is intended to provide a short description of the error. The `message` field is not intended to be parsed by the client or shown to the user. It is intended as a developer hint to help the client developer debug the error.
+
+## Overview
```

Comment:
Refunds are explained in case the channel open failed. This is the most important failure scenario IMO. I don't see any other scenario that is important enough to include here. Please point to one if you think it I should include it.